version.php

```diff
@@ -26,7 +26,7 @@
 
 /** @var stdClass $plugin */
 $plugin->component = 'block_sharing_cart';
-$plugin->version   = 2022050300;
+$plugin->version   = 2022050301;
 $plugin->requires  = 2021051700; // Moodle 3.11
-$plugin->release   = '3.11, release 1';
+$plugin->release   = '3.11, release 2';
```

Comment:
The same reason in the [README.md](https://github.com/praxisdigital/moodle-block_sharing_cart/blob/67a592a18fcfd08cf7e6c1c99485f33470a67c82/README.md)

Reply:
Changes added